File: 백준/Silver/4134. 다음 소수/다음 소수.py
t = int(input())

def is_prime(n):
    if n == 0 or n == 1:
        return 0
    for i in range(2, int(n ** 0.5) + 1):
        if n % i == 0:
            return 0
    return 1

for _ in range(t):
    n = int(input())
    
    i = n
    while 1:
        if is_prime(i) == 1:
            print(i)
            break
        i += 1

File: 백준/Silver/4134. 다음 소수/다음 소수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int T = sc.nextInt();
        for (int c = 0; c < T; c++) {
            long n = sc.nextLong();
            
            while (true) {
                if (isPrime(n)) break;
                n++;
            }
            
            System.out.println(n);
        }
    }
    
    private static boolean isPrime(long n) {
        if (n == 0 || n == 1) return false;
        for (int i = 2; i <= (int) Math.sqrt(n); i++) {
            if (n % i == 0) return false;
        }
        return true;
    }
}